src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2717,7 +2717,7 @@ static void window_ride_vehicle_mousedown(rct_window* w, rct_widgetindex widgetI
             window_ride_show_vehicle_type_dropdown(w, &w->widgets[widgetIndex]);
             break;
         case WIDX_VEHICLE_TRAINS_INCREASE:
-            if (ride->num_vehicles < MAX_VEHICLES_PER_RIDE)
+            if (ride->num_vehicles < OpenRCT2::Limits::MaxTrainsPerRide)
                 ride->SetNumVehicles(ride->num_vehicles + 1);
```

Comment:
This illustrates that `vehicle` is very ambiguous, sometimes referring to cars and sometimes to trains. We should make an issue to rename methods and fields to be unambiguous.